### PR TITLE
Upgrade dependencies, test on lowest dependencies, compatibility with doctrine 2.5.*

### PR DESCRIPTION
**Pull request by chives**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,10 +7,14 @@ php:
   - 5.6
   - hhvm
 
+env:
+  - COMPOSER_FLAGS=""
+  - COMPOSER_FLAGS="--prefer-lowest"
```
Comment (szymach) [inline .travis.yml]: skąd to powtórzenie?

**Comment by JarekW** [comment]
:+1:

**Comment by chives** [comment]
@fsi-open

**Comment by szymach** [comment]
:+1:

**Comment by rn0** [comment]
:+1:

**Pull request merged**